conda/activate.py

```diff
@@ -111,6 +119,8 @@ def __init__(self, shell, arguments=None):
             self.export_var_tmpl = 'set -gx %s "%s"'
             self.run_script_tmpl = 'source "%s"'
 
+            self.deactivatepost_tmpl = """eval $_CONDA_EXE shell.fish deactivatepost"""
```

Comment:
For `fish` we need two `eval`s, acc. to https://github.com/conda/conda/blob/4.5.0rc0/conda/shell/etc/fish/conf.d/conda.fish#L92.